priv/repo/migrations/20190911151157_add_works_table.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Meadow.Repo.Migrations.AddWorksTable do
+  use Ecto.Migration
+
+  def change do
+    create table("works", primary_key: false) do
+      add :id, :binary_id, null: false, primary_key: true
```

Comment:
I *think* that you don't need this here, because we added `migration_primary_key: [name: :id, type: :binary_id]` to the config file.